cli/client.go

```diff
@@ -385,6 +385,35 @@ func (c *viamClient) organizationLogoSetAction(cCtx *cli.Context, orgID, logoFil
 	return nil
 }
 
+type organizationsLogoGetArgs struct {
+	OrgID string
+}
+
+// OrganizationsLogoGetAction corresponds to `organizations logo get`.
+func OrganizationsLogoGetAction(cCtx *cli.Context, args organizationsLogoGetArgs) error {
+	c, err := newViamClient(cCtx)
+	if err != nil {
+		return err
+	}
+	return c.organizationsLogoGetAction(cCtx, args.OrgID)
+}
+
+func (c *viamClient) organizationsLogoGetAction(cCtx *cli.Context, orgID string) error {
+	if err := c.ensureLoggedIn(); err != nil {
+		return err
+	}
+
+	resp, err := c.client.OrganizationGetLogo(cCtx.Context, &apppb.OrganizationGetLogoRequest{
```

Comment:
check orgID is not empty?

Reply:
we already do this on the server side so omitting the duplicated logic here

Reply:
I still kind of think it should be handled here because the error message is clearer and nicer (doesn't have that annoying RPC error stuff)

Reply:
yeah thats fair. i general i try not to deduplicate the validation between the server + client because then the two can be out of sync. this feels pretty harmless and is in line with what we already have and adds readability so ill add it.